### PR TITLE
Add separate MathML annotation pages

### PR DESCRIPTION
**Pull request by pepelsbey**
### Description

MathML `<annotation>` and `<annotation-xml>` are now separated from the `<semantics>` element reference.

### Motivation

- Offload `<semantics>` element reference from annotations description and demos
- Improve annotation elements’ discoverability

### CODE REVIEW

File: files/en-us/web/mathml/element/annotation-xml/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: <annotation-xml>
+slug: Web/MathML/Element/annotation-xml
+page-type: mathml-element
+browser-compat: mathml.elements.annotation-xml
+---
+
+{{MathMLRef}}
+
+The **`<annotation-xml>`** [MathML](/en-US/docs/Web/MathML) element contaiting an annotation to the MathML expression in {{glossary("XML")}} format, for example [Content MathML](https://en.wikipedia.org/wiki/MathML#Content_MathML) or {{glossary("SVG")}}.
```
Comment (Josh-Cena) [inline files/en-us/web/mathml/element/annotation-xml/index.md:10]:
```suggestion
The **`<annotation-xml>`** [MathML](/en-US/docs/Web/MathML) element contains an annotation to the MathML expression in the {{glossary("XML")}} format, for example [Content MathML](https://en.wikipedia.org/wiki/MathML#Content_MathML) or {{glossary("SVG")}}.
```

File: files/en-us/web/mathml/element/annotation-xml/index.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+> [!NOTE]
+> Annotations are not supposed to be rendered by browsers and are hidden by default. Although Firefox and Safari render some annotation formats if the first child of the {{mathmlelement("semantics")}} element contains errors or missing.
```
Comment (Josh-Cena) [inline files/en-us/web/mathml/element/annotation-xml/index.md:13]:
```suggestion
> Annotations are not supposed to be rendered by browsers and are hidden by default. However, Firefox and Safari render some annotation formats if the first child of the {{mathmlelement("semantics")}} element contains errors or is missing.
```

File: files/en-us/web/mathml/element/annotation/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+title: <annotation>
+slug: Web/MathML/Element/annotation
+page-type: mathml-element
+browser-compat: mathml.elements.annotation
+---
+
+{{MathMLRef}}
+
+The **`<annotation>`** [MathML](/en-US/docs/Web/MathML) element contaiting an annotation to the MathML expression in a textual format, for example [LaTeX](https://www.latex-project.org/).
```
Comment (Josh-Cena) [inline files/en-us/web/mathml/element/annotation/index.md:10]:
```suggestion
The **`<annotation>`** [MathML](/en-US/docs/Web/MathML) element contains an annotation to the MathML expression in a textual format, for example [LaTeX](https://www.latex-project.org/).
```

File: files/en-us/web/mathml/element/annotation/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+> [!NOTE]
+> Annotations are not supposed to be rendered by browsers and are hidden by default. Although Firefox and Safari render some annotation formats if the first child of the {{mathmlelement("semantics")}} element contains errors or missing.
```
Comment (Josh-Cena) [inline files/en-us/web/mathml/element/annotation/index.md:13]:
```suggestion
> Annotations are not supposed to be rendered by browsers and are hidden by default. However, Firefox and Safari render some annotation formats if the first child of the {{mathmlelement("semantics")}} element contains errors or is missing.
```

**Review by Josh-Cena — COMMENTED** [review]
Just reviewing for English. @fred-wang can do the final review. Also linking https://github.com/mdn/content/pull/20472; you need to remove the redirect to pass CI. (Same comment for https://github.com/mdn/content/pull/36018)

File: files/en-us/web/mathml/element/semantics/index.md [hunk 5]
```diff
@@ -10,10 +10,16 @@ browser-compat: mathml.elements.semantics
 The **`<semantics>`** [MathML](/en-US/docs/Web/MathML) element associates annotations with a MathML expression, for example its text source as a [lightweight markup language](https://en.wikipedia.org/wiki/Lightweight_markup_language) or mathematical meaning expressed in a special {{glossary("XML")}} dialect. Typically, its structure is:
 
 - a first child which is a MathML expression to be annotated.
-- subsequent `<annotation>` or `<annotation-xml>` elements, the latter being reserved for XML formats such as [OpenMath](https://en.wikipedia.org/wiki/OpenMath).
+- subsequent {{mathmlelement("annotation")}} or {{mathmlelement("annotation-xml")}} elements, the latter being reserved for XML formats such as [Content MathML](https://en.wikipedia.org/wiki/MathML#Content_MathML) or {{glossary("SVG")}}.
```
Comment (fred-wang) [inline files/en-us/web/mathml/element/semantics/index.md:13]: No strong opinion on that, but any reason to remove OpenMath here?

File: files/en-us/web/mathml/element/semantics/index.md [hunk 6]
```diff
@@ -53,9 +67,11 @@ By default, only the first child of the `<semantics>` element is rendered while
...
     </mrow>
 
-    <!-- Annotate with "Content MathML", a dedicated XML dialect to
-         express the meaning of mathematical formulas. -->
-    <annotation-xml encoding="MathML-Content">
+    <!--
+      Annotate with content MathML, a dedicated XML dialect
+      to express the meaning of mathematical formulas.
+    -->
+    <annotation-xml encoding="application/mathml-content+xml">
```
Comment (fred-wang) [inline files/en-us/web/mathml/element/semantics/index.md:74]: If we are removing "MathML-Content" and "MathML-Presentation" here, maybe also do the same in the definitions of the encoding attribute?

(FWIW, they are defined in https://w3c.github.io/mathml/#encoding-names)

**Review by fred-wang — APPROVED** [review]
Thanks!

File: files/en-us/web/mathml/element/semantics/index.md (hunk 6 above)
Comment (pepelsbey) [reply files/en-us/web/mathml/element/semantics/index.md:74]: Agree! I tried to normalize encoding value in all examples to media type.

File: files/en-us/web/mathml/element/semantics/index.md (hunk 5 above)
Comment (fred-wang) [reply files/en-us/web/mathml/element/semantics/index.md:13]: OK sounds good to me, let's do that. If there are people who are familar with OpenMath who think it's important, they can always add examples later.

**Pull request merged**